bentoml/server/__init__.py

```diff
@@ -70,16 +70,12 @@ def start_dev_server(
         marshal_proc.start()
 
         bento_service = load_from_dir(saved_bundle_path)
-        api_server = BentoAPIServer(
-            bento_service, port=api_server_port, enable_swagger=enable_swagger
-        )
-        api_server.start()
+        api_server = BentoAPIServer(bento_service, enable_swagger=enable_swagger)
```

Comment:
Do you think `start_dev_server` could follow the same pattern as `start_prod_server`?

Reply:
Yeah. But that should be done by another PR. It involves more than the prod server.